### PR TITLE
fix log handling for crd manager

### PR DESCRIPTION
**Pull request by reshnm**
**What this PR does / why we need it**:

Minor cleanup for the crd manager log handling.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```feature user
NONE
```

### CODE REVIEW

File: pkg/controllers/servicetargetconfigs/add.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ import (
 
 // AddControllerToManager adds the landscaperdeployments controller to the manager
```
Comment (achimweigel) [inline pkg/controllers/servicetargetconfigs/add.go:20]: Is this comment correct?

**Review by achimweigel — APPROVED** [review]
/lgtm

only a small question

File: pkg/controllers/servicetargetconfigs/add.go (hunk 1 above)
Comment (reshnm) [reply pkg/controllers/servicetargetconfigs/add.go:20]: Good catch, it is not.

Comment (reshnm) [inline pkg/controllers/servicetargetconfigs/add.go:20]:
```suggestion
// AddControllerToManager adds the controller to the manager
```

**Pull request merged**